components/eam/bld/configure

```diff
@@ -2537,11 +2538,14 @@ sub write_filepath
           print $fh "$camsrcdir/eam/src/physics/crm/sam/SGS_TKE\n";
 
           # advection scheme
-          my $crm_adv = $cfg_ref->get('crm_adv');
-          if ($crm_adv eq 'UM5') {
-              print $fh "$camsrcdir/eam/src/physics/crm/sam/ADV_UM5\n";
+          my $crm_adv = 'MPDATA';
+          if (defined $opts{'crm_adv'}) { my $crm_adv = $cfg_ref->get('crm_adv'); }
+          if ($crm_adv eq 'MPDATA') {
+            print $fh "$camsrcdir/eam/src/physics/crm/sam/ADV_MPDATA\n";
+          } elsif ($crm_adv eq 'UM5') {
+            print $fh "$camsrcdir/eam/src/physics/crm/sam/ADV_UM5\n";
```

Comment:
Guess you prefer clean readability. Otherwise the above if and elsif can be combined and use just 1 print $fh statement. Either way is good.

Reply:
yea, I don't really understand how to use lists in perl, otherwise I agree that it could be cleaner/shorter.

Reply:
Just an FYI and not really requesting a change: To combine, the if statement can be
`if ("MPDATA,UM5" =~ /$crm_adv/) {  ... }`  # $crm_adv be a substring of the list, which can be joined in any way.
or more Fortran-like way of  `if ($crm_adv eq "MPDATA" || $crm_adv eq "UM5") { ... }`